delta/src/main/scala/io/qbeast/spark/delta/DeltaMetadataWriter.scala

```diff
@@ -228,7 +228,8 @@ private[delta] case class DeltaMetadataWriter(
       .collect()
       .map(DeltaQbeastFileUtils.fromAddFile(dimensionCount))
       .flatMap(_.tryReplicateBlocks(deltaReplicatedSet))
-      .map(DeltaQbeastFileUtils.toAddFile(dataChange = false))
+      .map(DeltaQbeastFileUtils.toAddFile)
+      .map(addFile => addFile.copy(dataChange = false))
```

Comment:
you can do this operation in one single map:

```
.map(file => {
val addFile = DeltaQbeastFileUtils.toAddFile(file)
addFile.copy(dataChange = false)
}
```

Reply:
But here it's confusing because according to the interface, I would expect that the `IndexFile` would have the `dataChange` flag already parsed in `toAddFile`

Reply:
I agree this can be done in the same map. 

On the other hand, I couldn't test this particular method. I'm not sure which test is using it, or if it's even being used, since the method is related to 'replication,' and as far as I know, replication is not utilized. In any case, I forced it to be `false` because the default `dataChange` is always `true`.